.github/CODEOWNERS

```
@@ -1,5 +1,5 @@
 # The default code owners of the uagents repo.
-* @jrriehl @Archento @lrahmani
+* @jrriehl @Archento @lrahmani @qati
```

```suggestion
* @jrriehl @Archento @lrahmani

/libs/core/ @qati
```

I propose this scope as your PR only targets changes in this directory.